tools/releaseBuild/azureDevOps/templates/release-SDKTests.yml

```diff
@@ -35,6 +37,28 @@ jobs:
         artifact: metadata
         path: '$(Pipeline.Workspace)/releasePipeline/metadata'
 
+    - pwsh:  |
+        Import-Module "$(Build.SourcesDirectory)/build.psm1" -Force
+        New-NugetConfigFile -NugetFeedUrl $(PSInternalNugetFeed) -UserName $(PSInternalNugetFeedUserName) -ClearTextPAT $(PSInternalNugetFeedPAT) -FeedName AzDevOpsFeed -Destination '$(Build.SourcesDirectory)/test/hosting'
+
+        Get-ChildItem '$(Build.SourcesDirectory)/test/hosting'
+
+        # The above cmdlet creates a lower-case nuget.config. There also exists a NuGet.config which we needed to replace.
+        # Hence the following workaround
```

Comment:
This template will run on Windows, Linux and macOS. How would this be possible on Windows agent?

Reply:
This wouldn't but the move-item will be a no-op. I can add a if condition if you prefer.

Reply:
Added a condition